### ISSUE TITLE
Kubectl version causes an error and doesn't print client verison

### ISSUE THREAD

**Issue by darkn3rd** [kubernetes/kubectl#912]
**What happened**:

When doing `kubectl version` I get an error.  I do not know how to tell what version of the client I am running, and consequently, tools that depend on this, like `eksctl` fail.

```
error: Missing or incomplete configuration info.  Please point to an existing, complete config file:

  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
````

**What you expected to happen**:

I expected that the client version would be printed out, so I know what version of the client I am running.   If a cluster has not been provisioned before, then `kubectl` should fall back and fetch just the client.  A warning can be produced but should print at least the client, not fail for client information.   I hope this pattern can be changed for usability.

**How to reproduce it (as minimally and precisely as possible)**:

```bash
kubectl version
```

```yaml
# ~/.kube/config
apiVersion: v1
clusters: []
contexts: []
current-context: ""
kind: Config
preferences: {}
users: []
```

**Anything else we need to know?**:

Getting the version of the client tool installed should not be broken or have dependencies such as a configuration.  

**Environment**:
- Kubernetes client and server versions (use `kubectl version`):
```
error: Missing or incomplete configuration info.  Please point to an existing, complete config file:

  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
```
- Cloud provider or hardware configuration: n/a
- OS (e.g: `cat /etc/os-release`):
```
NAME="Pop!_OS"
VERSION="19.10"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.10"
VERSION_ID="19.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=eoan
UBUNTU_CODENAME=eoan
LOGO=distributor-logo-pop-os
```

**Comment by michaelbeaumont** [comment]
I believe this is a duplicate of https://github.com/kubernetes/kubectl/issues/853

**Comment by eddiezane** [comment]
@darkn3rd thanks for the detailed report!

Echoing @michaelbeaumont, this should be fixed in kubectl 1.18+ if you upgrade.

/triage duplicate
/close
